api/openapi.yaml

```diff
@@ -1958,29 +1958,43 @@ components:
           type: "input"
     IngestedEvent:
       description: An ingested event with optional validation error.
+      readOnly: true
       type: object
       additionalProperties: false
       required:
         - event
+        - ingestedAt
+        - createdAt
       properties:
         event:
           $ref: "#/components/schemas/Event"
         validationError:
           type: string
-          readOnly: true
           example: "invalid event"
+        ingestedAt:
+          type: string
+          format: date-time
+          description: The date and time the event was ingested.
+          example: "2024-01-01T00:00:00Z"
+        createdAt:
```

Comment:
Maybe call this stored_at (based on the description)

Reply:
I was thinking about it, but `created_at` feels like the standard for DB modelling.

Reply:
For a standard CRUD operation, sure. This isn't that. Even the description says it's the timestamp for storing the event. Lastly, it appears in the API as well. I would prioritize ubiquitous language that makes sense in the context over DB modeling lingo.

Reply:
Renamed it.